### Description of Issue/Question
Trying to run the `salt.modules.win_pkg` fails on `oxygen.rc1`.
See this:
```
    The minion function caused an exception: Traceback (most recent call last):
      File "c:\salt\bin\lib\site-packages\salt\minion.py", line 1624, in _thread_return
        return_data = minion_instance.executors[fname](opts, data, func, args, kwargs)
      File "c:\salt\bin\lib\site-packages\salt\executors\direct_call.py", line 12, in execute
        return func(*args, **kwargs)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 1142, in install
        old = list_pkgs(saltenv=saltenv, refresh=refresh, versions_as_list=True)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 400, in list_pkgs
        name_map = _get_name_map(saltenv)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 1846, in _get_name_map
        u_name_map[k.decode('utf-8')] = name_map[k]
      File "c:\salt\bin\lib\encodings\utf_8.py", line 16, in decode
        return codecs.utf_8_decode(input, errors, True)
    UnicodeEncodeError: 'ascii' codec can't encode characters in position 5-11: ordinal not in range(128)
```

### Setup
WIndows minion and linux master.

### Steps to Reproduce Issue
Try to run `salt.modules.win_pkg`commands from the master to the win minion and it will fail.
```
[root@localhost ~]# salt '*' pkg.install firefox
win:
    The minion function caused an exception: Traceback (most recent call last):
      File "c:\salt\bin\lib\site-packages\salt\minion.py", line 1624, in _thread_return
        return_data = minion_instance.executors[fname](opts, data, func, args, kwargs)
      File "c:\salt\bin\lib\site-packages\salt\executors\direct_call.py", line 12, in execute
        return func(*args, **kwargs)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 1142, in install
        old = list_pkgs(saltenv=saltenv, refresh=refresh, versions_as_list=True)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 400, in list_pkgs
        name_map = _get_name_map(saltenv)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 1846, in _get_name_map
        u_name_map[k.decode('utf-8')] = name_map[k]
      File "c:\salt\bin\lib\encodings\utf_8.py", line 16, in decode
        return codecs.utf_8_decode(input, errors, True)
    UnicodeEncodeError: 'ascii' codec can't encode characters in position 5-11: ordinal not in range(128)
[root@localhost ~]# salt '*' pkg.list_pkgs
win:
    The minion function caused an exception: Traceback (most recent call last):
      File "c:\salt\bin\lib\site-packages\salt\minion.py", line 1624, in _thread_return
        return_data = minion_instance.executors[fname](opts, data, func, args, kwargs)
      File "c:\salt\bin\lib\site-packages\salt\executors\direct_call.py", line 12, in execute
        return func(*args, **kwargs)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 400, in list_pkgs
        name_map = _get_name_map(saltenv)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 1846, in _get_name_map
        u_name_map[k.decode('utf-8')] = name_map[k]
      File "c:\salt\bin\lib\encodings\utf_8.py", line 16, in decode
        return codecs.utf_8_decode(input, errors, True)
    UnicodeEncodeError: 'ascii' codec can't encode characters in position 5-11: ordinal not in range(128)
[root@localhost ~]# salt '*' pkg.latest_version chrome
win:
    The minion function caused an exception: Traceback (most recent call last):
      File "c:\salt\bin\lib\site-packages\salt\minion.py", line 1624, in _thread_return
        return_data = minion_instance.executors[fname](opts, data, func, args, kwargs)
      File "c:\salt\bin\lib\site-packages\salt\executors\direct_call.py", line 12, in execute
        return func(*args, **kwargs)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 128, in latest_version
        versions_as_list=True, saltenv=saltenv, refresh=refresh)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 400, in list_pkgs
        name_map = _get_name_map(saltenv)
      File "c:\salt\bin\lib\site-packages\salt\modules\win_pkg.py", line 1846, in _get_name_map
        u_name_map[k.decode('utf-8')] = name_map[k]
      File "c:\salt\bin\lib\encodings\utf_8.py", line 16, in decode
        return codecs.utf_8_decode(input, errors, True)
    UnicodeEncodeError: 'ascii' codec can't encode characters in position 5-11: ordinal not in range(128)
[root@localhost ~]# 
```

### Versions Report
```
    Salt Version:
               Salt: 2018.2.0-227-gfcf8a36
     
    Dependency Versions:
               cffi: 1.10.0
           cherrypy: Not Installed
           dateutil: 2.6.1
          docker-py: Not Installed
              gitdb: 2.0.3
          gitpython: 2.1.7
              ioflo: Not Installed
             Jinja2: 2.10
            libgit2: Not Installed
            libnacl: 1.6.1
           M2Crypto: Not Installed
               Mako: 1.0.7
       msgpack-pure: Not Installed
     msgpack-python: 0.4.8
       mysql-python: Not Installed
          pycparser: 2.18
           pycrypto: 2.6.1
       pycryptodome: Not Installed
             pygit2: Not Installed
             Python: 2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:25:58) [MSC v.1500 64 bit (AMD64)]
       python-gnupg: 0.4.1
             PyYAML: 3.12
              PyZMQ: 16.0.3
               RAET: Not Installed
              smmap: 2.0.3
            timelib: 0.2.4
            Tornado: 4.5.2
                ZMQ: 4.1.6
     
    System Versions:
               dist:   
             locale: cp1252
            machine: AMD64
            release: 2016Server
             system: Windows
            version: 2016Server 10.0.14393  Multiprocessor Free
```